audio/deepvoice3/README.md

```diff
@@ -0,0 +1,67 @@
+# DeepVoice3
+
+## Description
+Deep learning model that performs end-to-end neural speech synthesis. Requires fewer parameters and is faster than other state-of-the-art neural speech synthesis systems.
+
+## Model
+
+|Model        |Download  | Download (with sample test data)|ONNX version|Opset version|Accuracy|
+|-------------|:--------------|:--------------|:--------------|:--------------|:--------------|
+|DeepVoice3      | [52.8 MB](audio/deepvoice3/model/deepvoice3-11.onnx) | [49.1 MB](audio/deepvoice3/model/deepvoice3-11.tar.gz) |1.6| 11|Speech naturalness of 95.5 Pronunciation accuracy of 95.1 |
```

Comment:
Accuracy portion renders weirdly. Are there actual metric names mentioned in the paper for "speech naturalness" and Pronunciation accuracy?

```suggestion
|DeepVoice3      | [52.8 MB](audio/deepvoice3/model/deepvoice3-11.onnx) | [49.1 MB](audio/deepvoice3/model/deepvoice3-11.tar.gz) |1.6| 11|Speech=95.5, Pronunciation=95.1 |
```

Reply:
I looked more into the paper and changed the scoring to Mean Opinion Score (MOS) of 3.78